index.js

```diff
@@ -2397,6 +2402,23 @@ window.onload = () => {
 'use strict'
 const { SHA3 } = require('sha3')
 
+/**
+ * Converts UTC midnight of local YYYY-MM-DD string to ms since 1970-01-01 UTC
```

Comment:
@cosmc this is somewhat hard to understand (the day displayed to the user is always local time, but the timestamp sent to the NIST server is UTC midnight) but it was by far the easiest to implement without any libraries